kjourney/src/UI.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
```

Comment:
UI.html? that name needs to change! same as before, why is this not part of a component?